### PR TITLE
Feedback banner

### PR DESCRIPTION
**Pull request by BNewing**
This adds in a banner, which you can toggle and edit via the WordPress admin settings. You can turn it on and off, and edit the content in it. This is so we can add a feedback banner, for the ALB refresh.

### CODE REVIEW

File: web/app/themes/lawcom/lib/utils.php [hunk 1]
```diff
@@ -27,3 +27,9 @@ function project_has_documents($projectId) {
 
   return $documents->have_posts();
 }
+
+// Tell WordPress to use searchform.php from the templates/ directory
+function get_feedback_banner() {
+  locate_template('/templates/feedback-banner.php', true, false);
+}
+add_filter('get_feedback_banner', 'get_feedback_banner');
```
Comment (RobLoweMOJ) [inline web/app/themes/lawcom/lib/utils.php:35]: Just curious why you made this a function? Also I dont think you need the add_filter.

Comment (BNewing) [reply web/app/themes/lawcom/lib/utils.php:35]: Good question! I was unsure about this bit - and if `utils.php` was the right place for it! What do you think? 

I copied the above code which pulls in the form - is there another way of doing it :D?

Comment (RobLoweMOJ) [reply web/app/themes/lawcom/lib/utils.php:35]: I would just use get_template_part where your calling get_feedback_banner

Comment (BNewing) [reply web/app/themes/lawcom/lib/utils.php:35]: Ok! I'll switch to that :D is it alright in `utils.php`?

Comment (BNewing) [reply web/app/themes/lawcom/lib/utils.php]: I've removed it from `utils` and put it straight into the `header.php` file instead :D

**Comment by RobLoweMOJ** [comment]
Looks good.

**Pull request merged**